scripts/travis/release_packages.sh

```diff
@@ -50,9 +52,10 @@ function promote_stable() {
     init_s3cmd
 
     # Copy the _CHANNEL_ pending 'node' files to _CHANNEL-canary_
```

Comment:
Update comment

Reply:
What comment update is needed?

Reply:
It's also copying the files to a new bucket
> Remove pending_ from 'node' files, rename to _CHANNEL-canary_, and copy to the RELEASE_BUCKET